graphql/resolvers.go

```diff
@@ -631,6 +633,39 @@ func (r *queryResolver) Task(ctx context.Context, taskID string, execution *int)
 	return apiTask, err
 }
 
+func (r *queryResolver) TaskAllExecutions(ctx context.Context, taskID string) ([]*restModel.APITask, error) {
+	latestTask, err := task.FindOneId(taskID)
+	if err != nil {
+		return nil, ResourceNotFound.Send(ctx, err.Error())
+	}
+	if latestTask == nil {
+		return nil, errors.Errorf("unable to find task %s", taskID)
+	}
+	allTasks := []*restModel.APITask{}
+	for i := 0; i < latestTask.Execution; i++ {
+		var dbTask *task.Task
+		dbTask, err = task.FindByIdExecution(taskID, &i)
+		if err != nil {
+			return nil, ResourceNotFound.Send(ctx, err.Error())
```

Comment:
including additional text aside from err.Error() can make it easier for us to debug queries because we can easily distinguish between errors propagated from line 639 and 649

Reply:
I think this is fine as is - both those errors are the same, which for the most part will be if we can't contact the db